README.md

```diff
@@ -62,6 +62,24 @@ At the moment, only source installation is supported. Use Ubuntu Focal.
     colcon build
     ```
 
+### Using docker
+#### Requirements
+1. Install Docker using [installation instructions.](https://docs.docker.com/engine/install/ubuntu/)
```

Comment:
We also need `nvidia-docker`: https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker